presto-elasticsearch/src/test/java/io/prestosql/elasticsearch/BaseElasticsearchSmokeTest.java

```diff
@@ -27,7 +27,6 @@
 import org.elasticsearch.client.RestClient;
```

Comment:
This commit needs to come after (or be squashed with) the one that fixes the bug

Reply:
Order is fine now (it was not before). Github UI ordering is broken, though.

Reply:
https://gist.github.com/findepi/7abf17c8a26d3b74e8bb9527caadfdbe

Reply:
Yeah - I know. I did not want to kill automation.